internal/exec/stages/files/passwd.go

```diff
@@ -30,6 +30,19 @@ func (s stage) createPasswd(config types.Config) error {
 		return fmt.Errorf("failed to create users: %v", err)
 	}
 
+	// to be safe, just blanket mark all passwd-related files rather than
+	// trying to make it more granular based on which executables we ran
+	if len(config.Passwd.Groups) != 0 || len(config.Passwd.Users) != 0 {
+		s.relabel(
+			"/etc/passwd*",
+			"/etc/group*",
+			"/etc/shadow*",
+			"/etc/gshadow*",
+			"/etc/.pwd.lock",
+			"/home",
```

Comment:
We'll need this to be `/var/home` on ostree-based systems right?  `restorecon` doesn't follow symlinks.

Though...I honestly still wonder if it wouldn't be simpler to think about deferring some of the Ignition work until post-boot, exactly things like `useradd`.  We talked about that somewhere, I'm not finding it at the moment.

Reply:
There's an equivalence rule that makes `/var/home` and `/home` the same.

Reply:
As for deferring... yeah that would fix it as well. But then we'd be "leaking out" even more from the initramfs than we already need to, and it'd be unconditional even when not using SELinux, right?

I think this would also break configs which want to place files in the created user's home dir. So we'd have to defer that, too. Not sure how common a scenario that is.

Reply:
(We're not at that ideal now, and never have been.)